src/test/java/com/xellitix/jenkins/plugintool/cli/command/FetchInstalledPluginsCommandTest.java

```diff
@@ -38,6 +41,15 @@ public void getName__Test() {
         .isEqualTo(NAME);
   }
 
+  @Test
+  public void getApiToken__Test() {
+    Optional<String> var = command.getApiToken();
```

Comment:
The variable is no longer needed since you figured out how to chain all of the assertion calls.